src/lib/falcor/FalcorRouter.js

```diff
@@ -1,4 +1,21 @@
 import BaseRouter from "falcor-router"
+import { ghostArticleQuery } from 'lib/ghostAPI'
+import { SQLArticleQuery, SQLAuthorQuery } from 'lib/SQL'
+import falcor from 'falcor'
+
+var $ref = falcor.Model.ref;  
+
+// Transform ghost API names into our names
+var ghostify = (name) => {
```

Comment:
Why do we need different names from the ghostApi? Also ghostify is ambiguous, mapGhostNames makes more sense.